README.rst

```diff
@@ -147,7 +152,7 @@ Windows users can also install HTTPie with `Chocolatey <https://chocolatey.org>`
 
 .. code-block:: bash
 
-    $ choco install httpie
+    $ choco upgrade httpie
```

Comment:
Note: `upgrade` will perform the installation if necessary. It is the same approach as `python -m pip install --upgrade`.